include/llvm/Analysis/DxilValueCache.h

```diff
@@ -51,8 +51,7 @@ struct DxilValueCache : public ImmutablePass {
   };
 
 private:
-
-  WeakValueMap ValueMap;
```

Comment:
Hides the `ValueMap` above.

Reply:
Looks like this was taken care of with a98b242cacfb42c0277c5eabe876fbb47a885f60.